src/screens/Send/SendScreen.tsx

```diff
@@ -144,6 +148,18 @@ function SendScreen({ route }) {
     });
   };
 
+  const getWalletIcon = (wallet) => {
```

Comment:
Please use if else here. This is not readable 
`const getWalletIcon = (wallet) => {
  if (wallet.entityKind === EntityKind.VAULT) {
    return wallet.type === VaultType.COLLABORATIVE ? <CollaborativeIcon /> : <VaultIcon />;
  } else {
    return <WalletIcon />;
  }
};`